manifests/charts/istio-control/istio-discovery/files/gen-istio.yaml

```diff
@@ -1605,17 +1606,17 @@ spec:
       patch:
         operation: INSERT_BEFORE
         value:
-          name: istio.stats
+          name: envoy.filters.network.wasm
```

Comment:
@bianpengyuan please fix this in a subsequent PR, we don’t want to use the old style names in 1.6.
@kyessenov are free form names not supported by tcp filters yet?

Reply:
Created issue to track this: https://github.com/istio/istio/issues/23277

Reply:
Semantic does not work with this sha update. Not sure if it because of upstream bug or the necessary PR is not in 1.14.1.

Reply:
It is in use in this PR under http connection manager, so works there but not for tcp?

Reply:
Yep only network filter.

Reply:
Ok, thanks for opening the linked issue.